chcemvediet/apps/inforequests/tests/__init__.py

```diff
@@ -207,7 +106,7 @@ def _create_inforequest_scenario__action(self, inforequest, branch, args):
                 else:
                     default_mail_type = Message.TYPES.INBOUND
                     default_rel_type = InforequestEmail.TYPES.OBLIGEE_ACTION
-                    default_from_name, default_from_mail = next(branch.obligee.emails_parsed)
+                    default_from_name, default_from_mail = next(iter(branch.obligee.emails_parsed))
```

Comment:
budeme to potrebovat aby sme nam presili niektore testy v test_actions

Reply:
A nestaci proste `branch.obligee.emails_parsed[0]`?

Reply:
Nie, problem je ze niekedy bolo `.emails_parsed` type `list` a momentalne to je `tuple`. A `next` nespolupracuje spolu s tuple. `TypeError: tuple object is not an iterator`

Reply:
@viliambalaz `Obligee.emails_parsed` je list, ved vracia list, konkretne list tuplov:
```
return [(n, a) for n, a in getaddresses([self.emails]) if a]
```
Kedysi to bol asi iterator, nie list. Tak vtedy sa pouzil `next` na precitanie prveho prvku. Teraz, ked to je list, tak sa da napisat:

```
default_from_name, default_from_mail = branch.obligee.emails_parsed[0]
```

Reply:
@martinmacko47 Pardon, ano opravujem sa, ide to tak. :ok: